tcms/testplans/templates/testplans/get.html

```diff
@@ -19,6 +19,7 @@ <h1 class="col-md-12" style="margin-top: 0">
               data-perm-remove-testcase="{{ perms.testcases.delete_testcaseplan }}"
               data-perm-add-testcase="{{ perms.testcases.add_testcase }}"
               data-trans-default-tester-prompt-message="{% trans 'Please type new email or username' %}"
+              data-trans-no-testcases-selected="{% trans 'No test cases selected! Please select at least one test case!'%}"
```

Comment:
Please change to "No rows selected! Please select at least one!"  so we can start unifying these messages and reduce the translation burden a bit.

Reply:
I looked over testrun branch, there the message is "No executions selected! Please select at least one execution!".

Reply:
Yes I know. You copied correctly but the message should be the same in both branches and everywhere else where there is an error message for multiple selection. That's why it needs to be changed.